collated/main/A0139916U.md

```diff
@@ -0,0 +1,2151 @@
+# A0139916U
+###### \java\seedu\savvytasker\commons\events\model\AliasSymbolChangedEvent.java
+``` java
+package seedu.savvytasker.commons.events.model;
+
+import seedu.savvytasker.commons.events.BaseEvent;
+import seedu.savvytasker.model.alias.AliasSymbol;
+
+/**
+ * Represents an event where the user has added an alias symbol.
+ */
+public class AliasSymbolChangedEvent extends BaseEvent {
+    /**
+     * The action that was performed, i.e. the symbol was added or removed.
+     */
+    public enum Action {
+        Added,
+        Removed;
+    }
+    
+    public final AliasSymbol symbol;
```

Comment:
Is `public` the best access modifier for `symbol` ? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.